__tests__/app.test.js

```diff
@@ -78,3 +78,38 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200: returns all the articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
```

Comment:
we should also check that there isn't a body - at the moment if there is, this would still pass due to the flexible matcher toMatchObject() used here